app/odk.py

```diff
@@ -38,33 +39,75 @@ def update_review_state(project_id, form_id, submission_id, review_state):
     except AssertionError:
         log.error("Error while update submission state")
         print(review_submission_response.status_code)
-    client.close()
-
+    
 
 
 
+def write_users():
+    file = open('users.csv', 'w')
```

Comment:
Utiliser un contexte manager évite de faire le open / close : https://www.geeksforgeeks.org/with-statement-in-python/
```
with open('users.csv', 'w') as file:
    blah blah
```